### DISCUSSION TITLE
Pretix 3.15: Is it safe to ignore "MySQL does not support a database index on longtext columns."?

### DISCUSSION THREAD

**Discussion by seybsen** [pretix/pretix#1988] [Q&A] [answered]
Hi,

after I upgraded my pretix [small scale deployment](https://docs.pretix.eu/en/latest/admin/installation/docker_smallscale.html) to 3.15 I'm getting following message when the [cronjob](https://docs.pretix.eu/en/latest/admin/installation/docker_smallscale.html#cronjob) runs. Is it safe to ignore this? If so, how to silence this warning without silencing possibly important messages with e.g. `>/dev/null 2>&1`?

```
System check identified some issues:

WARNINGS:
pretixbase.RevokedTicketSecret.secret: (fields.W162) MySQL does not support a database index on longtext columns.
        HINT: An index won't be created. Silence this warning if you don't care about it.
```

MySQL Version: 5.7.28
Installed Plugins: passbook, mollie, fontpack-free, sepadebit

I could need another HINT here :detective: :sweat_smile: 
Thanks! :slightly_smiling_face:

**Comment by raphaelm** [comment, marked as answer]
Yes, it is safe to ignore (it will just be slower in some rare use cases of pretix). We're trying to figure out if we can get rid of the warning.

**Reply by seybsen** [reply]
Thank you @raphaelm
